setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
I indeed set it up for PyWavelets, but I don't remember any particular problem with Cython that needed resolving beyond installing it via ``requirements.txt``. Certainly nothing like ``gcc`` being called ``x86_64-linux-gnu-gcc``. Sorry.